tests/test_hedgehog_smoke.py

```diff
@@ -215,6 +215,74 @@ def test_vrrp_smoke(setup):
                                                                                                 version=version,
                                                                                                 priority=priority))
 
+def check_container_restarts_helper(duthost, container):
```

Comment:
how about moving helper script at the end of the file?
propose to move it for more easy navigate through the file